README.md

```diff
@@ -30,7 +30,7 @@ You can find instructions on setting up the project locally below. To get a loca
 These tools required regardless of your system:
 
 * 64-bit [NodeJS](https://nodejs.org/en/download/) 12.x or higher
-* [Yarn](https://yarnpkg.com/getting-started/install#per-project-install)
+* [Yarn](https://yarnpkg.com/getting-started/install): `npm install --global yarn`
```

Comment:
Hey @Pospelove, I've changed this link a bit, because this actually leads to the part which says about adding yarn itself to source tree, while we don't do that.

@JustCaptcha Should we? (In a separate PR of course)

Reply:
lgtm